src/include/reconcile.h

```diff
@@ -224,8 +224,9 @@ struct __wt_reconcile {
      */
     WT_SAVE_UPD *supd; /* Saved updates */
     uint32_t supd_next;
+    uint32_t supd_restore_count; /* Number of restored updates */
     size_t supd_allocated;
-    size_t supd_memsize; /* Size of saved update structures */
+    size_t supd_memsize; /* Size of the saved updates' data (excluding the update structure) */
```

Comment:
Would it be more accurate to call this the `supd_restore_memsize`?

Reply:
Yes, I'll change the name.

Reply:
[ce8c34b](https://github.com/wiredtiger/wiredtiger/pull/8127/commits/ce8c34bc0d6a24e0b3163090f32645b1290d157d)